test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift

```diff
@@ -66,7 +66,7 @@ class ViewController: UIViewController, WKUIDelegate, WKNavigationDelegate, UIDo
     }
     
     func pickSnapshot() {
-        let picker = UIDocumentPickerViewController(documentTypes: ["public.data"], in: .open)
+        let picker = UIDocumentPickerViewController(documentTypes: ["com.bentley.bim-imodel"], in: .open)
```

Comment:
We purposely made this use public.data a while ago because certain devices have a bug where it will filter out everything, even .bim files and we cannot select anything.  I just saw this happen again, for example on my Ipad Pro 12.9" 1st Gen on a 2.19.x build of display-test-app, where it does not use public.data. @khanaffan has seen this as well I believe

Reply:
Are you sure that this isn't due to incorrect settings in Info.plist? Note that in addition to changing the documentTypes, I also changed the Info.plist's registration of com.bentley.bim-imodel. Is there any way you can test in 2.19 with this updated Info.plist?

Reply:
Very possible - would be great if this PR fixes that problem we've seen.  I'll build this branch shortly and verify it works on the device I've seen this problem with.

Reply:
I forgot in my post above that "com.bentley.bim-imodel" is not the same as what was used before, so if you test in 2.19, you'd also need to change whatever documentTypes it is using to be "com.bentley.bim-imodel".

Reply:
I'll just test this branch, based off 3.

Reply:
Tried it out -- it all works :+1: Thanks!